src/main/flight/pid.c

```diff
@@ -990,6 +991,22 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, const rollAndPitchT
             // Only increase ITerm if output is not saturated
             pidData[axis].I = ITermNew;
         }
+        #if defined(USE_TBH)
+            if (pidProfile->tbh_enabled){
+                // Non-linear Take-Back Half algorithm
+                // When the error term crosses through zero, this algorithm causes
+                // the I term to non-linearly jump to half-way between the current
+                // accumulated error and the accumulated error at the last zero
+                // crossing.
+                const float ThisSign = copysign(1.0, itermErrorRate); //-1 or 1
```

Comment:
`1.0` type is double, should be `1.0f`.
Also, we use `camelCase` for variables

Reply:
Good catch :)